tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/OrganisationControllerTests.cs

```diff
@@ -117,34 +117,11 @@ public async Task GetAllAsync_SingleOrganisationExists_ReturnsTheOrganisation(st
         [Test]
         public async Task GetAllAsync_ListOfOrganisationsExist_ReturnsTheOrganisations()
         {
-            var org1 = new Organisation
-            {
-                Id = Guid.NewGuid(),
-                Name = "Organisation 1",
-                OdsCode = "ODS 1",
-                PrimaryRoleId = "ID 1",
-                CatalogueAgreementSigned = false,
-                Address = _address1
-            };
-            var org2 = new Organisation
-            {
-                Id = Guid.NewGuid(),
-                Name = "Organisation 2",
-                OdsCode = "ODS 2",
-                PrimaryRoleId = "ID 2",
-                CatalogueAgreementSigned = true,
-                Address = _address2
-            };
-
-            var org3 = new Organisation
-            {
-                Id = Guid.NewGuid(),
-                Name = "Organisation 3",
-                OdsCode = "ODS 3",
-                PrimaryRoleId = "ID 3",
-                CatalogueAgreementSigned = true,
-                Address = null
-            };
+            var org1 = OrganisationBuilder.Create(1).WithAddress(_address1).Build();
+
+            var org2 = OrganisationBuilder.Create(2).WithCatalogueAgreementSigned(true).WithAddress(_address2).Build();
```

Comment:
There looks to be more calls to `WithCatalogueAgreementSigned(true)`. Would it simplify the code to switch the default to `true` & just create one organisation with a false Catalogue Agreement Signed?